moped-editor/src/auth/user.js

```diff
@@ -136,6 +137,8 @@ export const initializeUserDBObject = async (session) => {
       throw error;
     }
   }
+
+  return sessionDataFromLocalStorage;
```

Comment:
I didn't update this function to return the user data if it already existed in the local storage entry. So, we were overwriting the valid user data object with `undefined`.